docs/framework/release-notes/2024/04-april-security-and-quality-rollup.md

```diff
@@ -0,0 +1,103 @@
+# April 2024 Security and Quality Rollup
```

Comment:
```suggestion
# April 2024 security and quality rollup
```

Reply:
The docs style is to use "Sentence casing" in titles, where only the first word is capitalized.